nemo/deploy/nlp/query_llm.py

```diff
@@ -141,7 +141,7 @@ def query_llm(
                     "object": "text_completion",
                     "created": int(time.time()),
                     "model": self.model_name,
-                    "choices": [{"text": str(sentences)}],
+                    "choices": [{"text": sentences}],
```

Comment:
Projecting NumPy array to string makes it hard to recover it later for further processing (note missing comma `","` after the first element):
```
str(np.array([["a"], ["b"]])) == "[['a']\n ['b']]"
```